pyproject.toml

```diff
@@ -25,32 +26,33 @@ classifiers = [
     "Operating System :: Unix",
     "Operating System :: MacOS",
 ]
-requires-python = ">=3.9"
-dependencies = [
-    "aws-cdk-lib==2.48.0",
-    "aws-cdk.aws-lambda-python-alpha",
-    "constructs>=10.0.0,<11.0.0",
-    "boto3",
-    "opensearch-py",
-    "requests",
-    "python-jose"
-]
 
-[project.optional-dependencies]
-dev = [
-    "pytest==6.2.5",
-    "moto"
-]
+[tool.poetry.dependencies]
+python = ">=3.9,<4"
+# Required for aws-cdk.aws-lambda-python-alpha
+aws-cdk-lib = ">=2.66.0,<3.0.0"
+#aws-cdk-lib = "2.48.0"
+boto3 = "^1.26.78"
+"aws-cdk.aws-lambda-python-alpha" = "^2.66.0a0"
+constructs = ">=10.0.0,<11.0.0"
+black = "^23.1.0"
+
+[tool.poetry.group.tests.dependencies]
+# Installed by default, can be installed without these dependencies using `poetry install --without tests`
+pytest = "==6.2.5"
+moto = "^4.1.3"
+hypothesis = "^6.68.2"
+
+[tool.poetry.group.cdk]
+# Not installed by default, can be installed using `poetry install --with cdk`
+optional = true
+
+[tool.poetry.group.cdk.dependencies]
+opensearch-py = "^2.1.1"
+requests = "^2.28.2"
+python-jose = "^3.3.0"
 
 [project.urls]
 homepage = "https://github.com/IMAP-Science-Operations-Center"
-repository = "https://github.com/IMAP-Science-Operations-Center/SDS-in-a-box"
+repository = "https://github.com/IMAP-Science-Operations-Center/sds-data-manager"
 
-[tool.pytest.ini_options]
```

Comment:
We should keep all of these other tool sections as they are unrelated to the poetry work. We also need the marker and tests path to be discovered.

Reply:
Ok, good catch. I tried to keep the sections but must have deleted them at some point while formatting everything.